doc/layout-config.rst

```diff
@@ -59,6 +59,9 @@ options:
 ``offset`` (integer/string)
    The offset of a partition.
 
+``align`` (integer/string)
+   Aligns the size of the partition to the specified value. The default is 1KiB.
```

Comment:
I think we should be more precise regarding the default. It is actually two sectors, which in almost all cases is 1KiB (2 × 512B), but I still think we should mention it that way.